src/server/game/Spells/Spell.cpp

```diff
@@ -4291,60 +4291,93 @@ void Spell::WriteCastResultInfo(WorldPacket& data, Player* caster, SpellInfo con
     switch (result)
     {
         case SPELL_FAILED_REQUIRES_SPELL_FOCUS:
+        {
             data << uint32(spellInfo->RequiresSpellFocus);  // SpellFocusObject.dbc id
             break;
+        }
         case SPELL_FAILED_REQUIRES_AREA:                    // AreaTable.dbc id
+        {
             // hardcode areas limitation case
             switch (spellInfo->Id)
             {
                 case 41617:                                 // Cenarion Mana Salve
                 case 41619:                                 // Cenarion Healing Salve
+                {
                     data << uint32(3905);
                     break;
+                }
```

Comment:
I don't think we should require brackets in switch-case, but I'm not against it either :)

Reply:
I just saw someone that used it so I used it for all.
I also thought it was kinda weird

Reply:
Only needed if you define variables inside the case.